sdc/rabbit/consumers.py

```diff
@@ -100,25 +101,6 @@ def add_on_connection_close_callback(self):
         logger.info('Adding connection close callback')
         self._connection.add_on_close_callback(self.on_connection_closed)
```

Comment:
Is this function still required?

Reply:
oooh good spot.  I think this can be removed